tests/runtime/device_test.py

```diff
@@ -151,20 +151,28 @@ def testFromTorchDevice(self):
         print(device.dump_device_info())
 
     def testJit(self):
-        from shark_turbine.ops import iree as iree_ops
+        from shark_turbine.ops import _str_format_test_ops as test_ops
```

Comment:
NIT: Can do later or set as TODO. Add test for jinja-based formatting and check output MLIR string from export perhaps. :)

Reply:
There is one below on the CPU. I debated having more of a lower level test but in this case, I think a functional test gets the job done.